testing/trino-testing/src/main/java/io/trino/testing/BaseFaultTolerantExecutionConnectorTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public abstract class BaseFaultTolerantExecutionConnectorTest
```

Comment:
This is not used as a baseclass. If test structure is to stay as in this PR then this class should be renamed to `FaultTolerantExecutionConnectorTestHelper` or sth and just have `public static Map<String, String> getExtraProperties()` method.

Reply:
The idea was that other connectors (than Hive) might want to directly extend the `BaseFaultTolerantExecutionConnectorTest` if there's no connector specific tests need to be implemented. However other connectors are also likely to have connector specific tests. So let me change it to be an utility class.